pkgs/applications/video/animdl/default.nix

```diff
@@ -0,0 +1,69 @@
+{ lib,
+  python3Packages,
+  buildPythonApplication,
+  fetchFromGitHub,
+  poetry-core,
+  anchor-kr,
+  anitopy,
+  click,
+  cssselect,
+  httpx,
+  lxml,
+  packaging,
+  pkginfo,
+  pycryptodomex,
+  pyyaml,
+  regex,
+  rich,
+  tqdm,
+  yarl
+}:
+buildPythonApplication rec {
+  pname = "animdl";
+  version = "1.7.24";
+
+  src = fetchFromGitHub {
+    owner = "justfoolingaround";
+    repo = "${pname}";
```

Comment:
```suggestion
    repo = "animdl";
```

Reply:
Would you mind explaining to me why? I've seen this in many other derivations

Reply:
it's bad practice and semanticlaly repo name is not the package name

Reply:
Understood